1.9/administration/upgrading.md

```diff
@@ -69,6 +69,12 @@ Proceed with upgrading every master node one-at-a-time in any order using the fo
     $ sudo bash ./dcos_node_upgrade.sh
     ```
 
+1.  Verify that the upgrade script succeeded and exited with the status code `0`:
+    ```
```

Comment:
Not used in the shell excerpts immediately beside it. Deferring to docs maintainers here.